### PR TITLE
💄 Use ES6; clean up CoffeeScript conversion in BrowserWindow module

### PR DESCRIPTION
**Pull request by stevekinney**
Another quick clean up to remove some of the extra cruft that came from converting from CoffeeScript to JavaScript.

### CODE REVIEW

File: lib/browser/api/browser-window.js [hunk 1]
```diff
@@ -74,109 +72,85 @@ BrowserWindow.prototype._init = function () {
...
-  for (i = 0, len = windows.length; i < len; i++) {
-    window = windows[i]
-    if (window.isFocused()) {
-      return window
-    }
+BrowserWindow.getFocusedWindow = () => {
+  for (let window of BrowserWindow.getAllWindows()) {
+    if (window.isFocused()) return window
   }
-  return null
```
Comment (zcbenz) [inline lib/browser/api/browser-window.js]: This is actually needed since this function needs to return null when there is no focused window.

Comment (stevekinney) [inline lib/browser/api/browser-window.js]: @zcbenz Ah, totally makes sense. I added it back.

**Comment by zcbenz** [comment]
This PR is mixed with other commits, and I'm unable to find out which ones are yours, do you mind doing a rebase to make this PR only include your commits?

**Comment by stevekinney** [comment]
Whoops—sorry I had a rebase mix-up. It should be better after the CI finishes.

**Comment by zcbenz** [comment]
👍

**Pull request merged**